internal/dashboard/auth_middleware.go

```diff
@@ -162,7 +164,10 @@ func (s *Server) userAuthenticationMiddleware(next http.Handler) http.Handler {
 			} else {
 				// General User have access only to the own resources
 				log.Info("invalid user authentication: general user trying to access other's resource", "userid", caller.Name, "target", user.Name)
-				w.WriteHeader(http.StatusForbidden)
+				errorResponse := dashv1alpha1.ErrorResponse{
+					Message: "not authorized",
+				}
+				dashv1alpha1.EncodeJSONResponse(errorResponse, pointer.Int(http.StatusForbidden), w)
```

Comment:
> do not have permission -> not allowed

Users cannot access the unauthorized resource URLs from Dashboard GUI.
So I think message is not necessary.

```suggestion
				w.WriteHeader(http.StatusForbidden)
```